### PR TITLE
2495: Ting field search integration 

### PR DESCRIPTION
**Pull request by holt83**
http://platform.dandigbib.org/issues/2495

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 305c8df6fd19e725376d215e4e4207610a1e11f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 661854f155411f5c4145571a7468a8b65b99d21d
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: b298334597a60a72fe67de673018ddf42fd52cc9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c7686d3429c6cb59b0740321c847107c9be7bc61
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: ccf8934bcfe7f9c7b5f9b563b41ee95767ea41d8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 81f5c6f06d2e7d0eb87cf70c489b654deacc1b78
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 97577c9124d96b83f8d4d3b6f5bea8bfee2c6a92
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 513e5878b983002dcccc9d6aef45713dffa474f4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 7080956d4ece3b7bc785df3a6c4415d6ca18f559
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1814a57560ab02e22374ce7e3682cb47c00c516f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b0c292967697fe46d218326efd83af7c98907fee
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b7e8b7412e022bb9264d03cafc34805020378406
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 810789df6a4d10fe06de1395f645b4bc6ad357d9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 77c06fe33b11ca1f449ac78f63df7f6e256438f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: d407dcd331928d769c936205ac12fb1616a94226
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e540db4355b8d6ddccb8e5d0bf5835253545821a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 342e6e4d37e65c1a802bf74e4445d156476e4afe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0fbf4842bc64752fc3961042379303bccd264e20
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 70107a76313bc67b7dd5d5e2e2788f1934f099bc
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 9843eac2dabe3f4fbce0b9767bb099d8d2df34dd
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e287f53691861630b9e6d8e45272b9632534061c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 32447c0677d3a1b1319b71dd0a09b2074695d4d5
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 087240efd1b1b0afd573e1a28545c6b5267e1c59
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0120ad3be03bafa28b8a383f6ab93d721993e568
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a9f2134209726f03f6c6ea7ec32f29d1b437628b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 94053292069f021b03a9dde93f1347da8879084f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 95ebc968f912938c7b9de5e1301d7ce13c6e0ba4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 5dc2ef644e5d5e79ce1d2017622bf7cfd9dd829b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 3fbde1ecdfde25b06e3b798a0d2348dee8dc7cfe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2104ea249f470a15f47ca605bdae30ece0da5b2a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.class.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  function list_sort_options() {
+    return array(
+      'title' => t('Title'),
+      'name' => t('Name'),
+      'weight' => t('Weight'),
+      'exposed' => t('Exposed'),
+    );
+  }
+
+  function list_build_row($profile, &$form_state, $operations) {
```
Comment (cableman) [inline modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.class.php]: Missing function documentation?

File: modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.class.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      case 'weight':
+        $this->sorts[$name] = $weight;
+        break;
+      case 'exposed':
+        $this->sorts[$name] = $exposed;
+        break;
+    }
+  }
+
+  function list_table_header() {
```
Comment (cableman) [inline modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.class.php]: Missing function documentation?

File: modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+    'submit' => 'ting_field_search_profile_export_form_submit'
+  ),
+
+  'handler' => array(
+    'class' => 'ting_field_search_ctools_export_ui',
+    'parent' => 'ctools_export_ui',
+  ),
+);
+
+function ting_field_search_profile_export_form(&$form, &$form_state) {
```
Comment (cableman) [inline modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc]: Missing function documentation?

File: modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+    foreach ($profile->facets as $facet) {
+      $row = _ting_field_search_build_facet_row($index, $facet);
+      $row_elements[$index] = array(
+        'name' => &$row['data'][0]['data'],
+        'title' => &$row['data'][1]['data'],
+        'sorting' => &$row['data'][2]['data'],
+        'weight' => &$row['data'][3]['data'],
+      );
+      $rows[$index] = $row;
+      //_ting_field_search_prepare_facet_row($form['facets']['table'], $index, $facet);
```
Comment (cableman) [inline modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc]: Commented out code?

File: modules/ting_field_search/ting_field_search.admin.inc [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ *
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.admin.inc]: Remove enter!

File: modules/ting_field_search/ting_field_search.admin.inc [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+ * Admin page callbacks for the Ting field search module.
+ */
+
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.admin.inc]: Remove enter!

File: modules/ting_field_search/ting_field_search.admin.inc [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+function ting_field_search_settings_form($form, &$form_state) {
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.admin.inc]: Why this "empty" settings form?

File: modules/ting_field_search/ting_field_search.install [hunk 8]
```diff
@@ -0,0 +1,354 @@
+  <?php
+
+/**
+ * @file
+ *
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.install]: Remove enter!

File: modules/ting_field_search/ting_field_search.install [hunk 9]
```diff
@@ -0,0 +1,354 @@
...
+  if (ting_field_search_get_status()) {
+    variable_del('cache_class_cache_ting');
+  }
+}
+
+/**
+ * Implements hook_schema().
+ */
+function ting_field_search_schema() {
+  $schema['ting_field_search_profile'] = array(
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.install]: Missing init of $scheme variable:

$schema = array();

File: modules/ting_field_search/ting_field_search.install [hunk 10]
```diff
@@ -0,0 +1,354 @@
...
+    ->execute()
+    ->fetchAllAssoc('name');
+}
+
+/**
+ * Adds the new config field to the profile table.
+ * Adds new serial primary key to profile table.
+ * Update foreign key in facet table to point to the new profile primary key.
+ */
+function ting_field_search_update_7000() {
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.install]: Technically these update functions are not run on DDB sites as this is a new module? So should they be removed?

File: modules/ting_field_search/ting_field_search.module [hunk 11]
```diff
@@ -0,0 +1,957 @@
+<?php
+
+/**
+ * @file
+ *
```
Comment (cableman) [inline modules/ting_field_search/ting_field_search.module]: Remove enter!

File: modules/ting_field_search/ting_field_search.install (hunk 10 above)
Comment (holt83) [inline modules/ting_field_search/ting_field_search.install]: Some other libraries are using this module, so I thought I would leave them to provide an easy migration path for them. I'm not sure if they have up to date version of ting_field_search and have runned these updates

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 305c8df6fd19e725376d215e4e4207610a1e11f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 661854f155411f5c4145571a7468a8b65b99d21d
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: b298334597a60a72fe67de673018ddf42fd52cc9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c7686d3429c6cb59b0740321c847107c9be7bc61
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: ccf8934bcfe7f9c7b5f9b563b41ee95767ea41d8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 81f5c6f06d2e7d0eb87cf70c489b654deacc1b78
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 97577c9124d96b83f8d4d3b6f5bea8bfee2c6a92
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 513e5878b983002dcccc9d6aef45713dffa474f4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 7080956d4ece3b7bc785df3a6c4415d6ca18f559
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1814a57560ab02e22374ce7e3682cb47c00c516f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b0c292967697fe46d218326efd83af7c98907fee
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b7e8b7412e022bb9264d03cafc34805020378406
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 810789df6a4d10fe06de1395f645b4bc6ad357d9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 77c06fe33b11ca1f449ac78f63df7f6e256438f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: d407dcd331928d769c936205ac12fb1616a94226
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e540db4355b8d6ddccb8e5d0bf5835253545821a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 342e6e4d37e65c1a802bf74e4445d156476e4afe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0fbf4842bc64752fc3961042379303bccd264e20
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 70107a76313bc67b7dd5d5e2e2788f1934f099bc
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 9843eac2dabe3f4fbce0b9767bb099d8d2df34dd
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e287f53691861630b9e6d8e45272b9632534061c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 32447c0677d3a1b1319b71dd0a09b2074695d4d5
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 087240efd1b1b0afd573e1a28545c6b5267e1c59
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0120ad3be03bafa28b8a383f6ab93d721993e568
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a9f2134209726f03f6c6ea7ec32f29d1b437628b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 94053292069f021b03a9dde93f1347da8879084f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 95ebc968f912938c7b9de5e1301d7ce13c6e0ba4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 5dc2ef644e5d5e79ce1d2017622bf7cfd9dd829b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 3fbde1ecdfde25b06e3b798a0d2348dee8dc7cfe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2104ea249f470a15f47ca605bdae30ece0da5b2a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 305c8df6fd19e725376d215e4e4207610a1e11f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 661854f155411f5c4145571a7468a8b65b99d21d
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: b298334597a60a72fe67de673018ddf42fd52cc9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c7686d3429c6cb59b0740321c847107c9be7bc61
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: ccf8934bcfe7f9c7b5f9b563b41ee95767ea41d8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 81f5c6f06d2e7d0eb87cf70c489b654deacc1b78
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 97577c9124d96b83f8d4d3b6f5bea8bfee2c6a92
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 513e5878b983002dcccc9d6aef45713dffa474f4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 7080956d4ece3b7bc785df3a6c4415d6ca18f559
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1814a57560ab02e22374ce7e3682cb47c00c516f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b0c292967697fe46d218326efd83af7c98907fee
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b7e8b7412e022bb9264d03cafc34805020378406
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 810789df6a4d10fe06de1395f645b4bc6ad357d9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 77c06fe33b11ca1f449ac78f63df7f6e256438f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: d407dcd331928d769c936205ac12fb1616a94226
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e540db4355b8d6ddccb8e5d0bf5835253545821a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 342e6e4d37e65c1a802bf74e4445d156476e4afe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0fbf4842bc64752fc3961042379303bccd264e20
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 70107a76313bc67b7dd5d5e2e2788f1934f099bc
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 9843eac2dabe3f4fbce0b9767bb099d8d2df34dd
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e287f53691861630b9e6d8e45272b9632534061c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 32447c0677d3a1b1319b71dd0a09b2074695d4d5
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 087240efd1b1b0afd573e1a28545c6b5267e1c59
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0120ad3be03bafa28b8a383f6ab93d721993e568
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a9f2134209726f03f6c6ea7ec32f29d1b437628b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 94053292069f021b03a9dde93f1347da8879084f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 95ebc968f912938c7b9de5e1301d7ce13c6e0ba4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 5dc2ef644e5d5e79ce1d2017622bf7cfd9dd829b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 3fbde1ecdfde25b06e3b798a0d2348dee8dc7cfe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2104ea249f470a15f47ca605bdae30ece0da5b2a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 305c8df6fd19e725376d215e4e4207610a1e11f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 661854f155411f5c4145571a7468a8b65b99d21d
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: b298334597a60a72fe67de673018ddf42fd52cc9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c7686d3429c6cb59b0740321c847107c9be7bc61
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: ccf8934bcfe7f9c7b5f9b563b41ee95767ea41d8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 81f5c6f06d2e7d0eb87cf70c489b654deacc1b78
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 97577c9124d96b83f8d4d3b6f5bea8bfee2c6a92
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 513e5878b983002dcccc9d6aef45713dffa474f4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 7080956d4ece3b7bc785df3a6c4415d6ca18f559
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1814a57560ab02e22374ce7e3682cb47c00c516f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b0c292967697fe46d218326efd83af7c98907fee
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: b7e8b7412e022bb9264d03cafc34805020378406
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 810789df6a4d10fe06de1395f645b4bc6ad357d9
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 77c06fe33b11ca1f449ac78f63df7f6e256438f3
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: d407dcd331928d769c936205ac12fb1616a94226
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e540db4355b8d6ddccb8e5d0bf5835253545821a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 342e6e4d37e65c1a802bf74e4445d156476e4afe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0fbf4842bc64752fc3961042379303bccd264e20
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 70107a76313bc67b7dd5d5e2e2788f1934f099bc
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 9843eac2dabe3f4fbce0b9767bb099d8d2df34dd
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: e287f53691861630b9e6d8e45272b9632534061c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 32447c0677d3a1b1319b71dd0a09b2074695d4d5
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 087240efd1b1b0afd573e1a28545c6b5267e1c59
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 0120ad3be03bafa28b8a383f6ab93d721993e568
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a9f2134209726f03f6c6ea7ec32f29d1b437628b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 94053292069f021b03a9dde93f1347da8879084f
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 95ebc968f912938c7b9de5e1301d7ce13c6e0ba4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 5dc2ef644e5d5e79ce1d2017622bf7cfd9dd829b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 3fbde1ecdfde25b06e3b798a0d2348dee8dc7cfe
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2104ea249f470a15f47ca605bdae30ece0da5b2a
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting_field_search/ting_field_search.install (hunk 10 above)
Comment (cableman) [inline modules/ting_field_search/ting_field_search.install]: 👍

**Comment by cableman** [comment]
Reviewed and awaiting release

**Pull request merged**